src/main/java/com/avides/springboot/testcontainer/common/container/AbstractEmbeddedContainer.java

```diff
@@ -50,6 +50,8 @@ private static String getDockerHost()
         {
             Builder builder = DefaultDockerClientConfig.createDefaultConfigBuilder();
             Field declaredField = builder.getClass().getDeclaredField("dockerHost");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "dockerHost" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.avidesmedia.de/coding_rules#rule_key=squid%3AS1192)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "declaredField". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.avidesmedia.de/coding_rules#rule_key=squid%3AS1854)